### ISSUE TITLE
Feature Request - check mime_type?

### ISSUE THREAD

**Issue by dustymc** [ArctosDB/arctos#5476]
**Is your feature request related to a problem? Please describe.**

An extension of https://github.com/ArctosDB/arctos/issues/5475, many of the problematic documents claim to be something they are not. (text/html claiming to be image/jpeg, mostly). The existence of these data suggest other similar data. 

**Describe what you're trying to accomplish** 

Misclassified MPDs break Arctos; other flle type mismatches break viewers, players, etc.

**Describe the solution you'd like**

Check mimetype at creation/edit; check existing data.

**Describe alternatives you've considered**

Let things be broken

**Additional context**

https://arctos.database.museum/media/10583845 + 'view as document' -->  fail 

**Priority**

Normal/high

**Comment by dustymc** [comment]
No interest, tabling.
